Development/nmos-cpp-node/config.json

```
@@ -285,5 +292,79 @@
     // ocsp_request_max [registry, node]: timeout for interactions with the OCSP server
     //"ocsp_request_max": 30,
 
+    // authorization_address [registry, node]: IP address or host name used to construct request URLs for the Authorization API (if not discovered via DNS-SD)
+    //"authorization_address": ip-address-string,
+
+    // authorization_port [registry, node]: used to construct request URLs for the authorization server's Authorization API (if not discovered via DNS-SD)
+    //"authorization_port" 443,
+
+    // authorization_version [registry, node]: used to construct request URLs for Authorization API (if not discovered via DNS-SD)
+    //"authorization_version": "v1.0",
+
+    // authorization_selector [registry, node]: used to construct request URLs for the authorization API (if not discovered via DNS-SD)
+    //"authorization_selector", "",
+
+    // authorization_request_max [registry, node]: timeout for interactions with the Authorization API /certs & /token endpoints
+    //"authorization_request_max": 30,
+
+    // fetch_authorization_public_keys_interval_min/fetch_authorization_public_keys_interval_max [registry, node]: used to poll for Authorization API public keys changes; default is about one hour
+    // "Resource Servers (Nodes) SHOULD seek to fetch public keys from the Authorization Server at least once every hour. Resource Servers MUST vary their retrieval
+    // interval at random by up to at least one minute to avoid overloading the Authorization Server due to Resource Servers synchronising their retrieval time."
+    // See https://specs.amwa.tv/is-10/releases/v1.0.0/docs/4.1._Behaviour_-_Authorization_Servers.html#authorization-server-public-keys
+    //"fetch_authorization_public_keys_interval_min": 3600,
+    //"fetch_authorization_public_keys_interval_max": 3660,
+
+    // access_token_refresh_interval [node]: time interval (in seconds) to refresh access token from Authorization Server
+    // It specified the access token refresh period otherwise Bearer token's expires_in is used instead.
+    // See https://specs.amwa.tv/is-10/releases/v1.0.0/docs/4.4._Behaviour_-_Access_Tokens.html#access-token-lifetime
+    //"access_token_refresh_interval": -1,
+
+    // client_authorization [node]: whether clients should use authorization to access protected APIs
+    //"client_authorization": false,
+
+    // server_authorization [registry, node]: whether server should use authorization to protect its APIs
+    //"server_authorization": false,
+
+    // authorization_code_flow_max [node]: timeout for the authorization code workflow (in seconds)
```

RFC 8252 calls these `flows` rather than `workflows`

```suggestion
    // authorization_code_flow_max [node]: timeout for the authorization code flow (in seconds)
```